variable/technology/electricity-grid.yaml

```diff
@@ -1,25 +1,87 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+ Cost of Demand-not-served|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit:  €/MWh
+
+Demand-not-served|Electricity:
+   description: Demand-not-server per region (i.e. aggregate-regions, countries, nuts1, nuts2, nuts3, or nodes).
+   unit:  GWh
 
 Network|Electricity|Demand|Reserve|Frequency Containment:
-   description: demand in frequency containment reserve (FCR)            
+   description: demand in frequency containment reserve (FCR)
    unit: MW
-   
+
 Network|Electricity|Demand|Reserve|Automatic Frequency Restoration:
-   description: demand in automatic frequency restoration reserve (AFRR)             
+   description: demand in automatic frequency restoration reserve (AFRR)
    unit: MW
-   
+
 Network|Electricity|Demand|Inertia:
-   description: demand in inertia            
+   description: demand in inertia
    unit: second
-   
+
+# description of variables related to transmission lines
+Active Power Flow|Electricity|Transmission|Line:
+   description: Active power flow through a line
+   unit: GW
+
+Capacity|Electricity|Transmission|Line:
```

Comment:
Is this not the same than Maximum Flow?

Reply:
Not, this is the active power flow in a line as a result of the optimization process.

Reply:
@erikfilias I was referring to the Capacity|Electricity|Transmission|Line variable, not Active Power Flow which is clear

Reply:
Ah ok, I got it. I'll remove it.
Also, `Maximum Flow|Electricity|Grid` could be renamed as `Maximum Flow|Electricity|Transmission` to be more precise and avoid misunderstanding about Grid. 
It could be an option for another modeler that could insert  `Maximum Flow|Electricity|Distribution`.

And, another **issue** could be a wording related to units in power systems because some variables are in GWh, MWh, and futures ones could be in kWh as well as the use of parameters in **per-unit**.

Reply:
True. We shall use the same wording everywhere, either Grid or Transmission but not both. Transmission is ok for me 

regarding units, there had been a discussion about it, there is also the same in Final Energy where demand is expressed in eJ/yr by the IAM community, and in GWh/yr by the electricity models community. (you can recongnise which variables were included by Daniel and which were by myself from this .....). It is only a mater of very simple conversion though, and the platform will offer conversion services. the discussion was about allowing to include different units in the nomenclature description or not. 
I would propose that we open an issue related to this topic

Reply:
Yeah, I agree.  I think that you refer to #14 , no?
Maybe as a first step in these files related to electricity models, we can reach an agreement about using units.
For example, we can use the maximum units e.g. to use GW instead of MW in large scale power models 
In the same way, local models could propose the use of kW for its variables.

@danielhuppmann , any opinion?
Do you know what criteria are being considered in the platform to perform the conversion service?

Reply:
It seems that the usual in european / national models is
MW for single power plants descrption (like Max Active POwer)
MWh for time series
GW for total installed capacity
GWh for yearly energy demand
(at least this is what I usually see in publications and open datasets)

Reply:
I suggest to introduce input data and output results in MW for power, instead of GW. Even these units can be extremely high when introducing models dealing with local communities

Reply:
Ok, I got it. 
Thanks!